src/SonarScanner.MSBuild.Shim/AdditionalFilesService.cs

```diff
@@ -113,23 +110,27 @@ private static AdditionalFiles PartitionAdditionalFiles(List<string> allFiles, A
     }
 
     private static string[] GetTestExtensions(AnalysisProperties properties) =>
```

Comment:
You should make sure, that the filename check for tests also includes a StringComparison. I think it is safe to use OrdinalIgnoreCase everywhere (also on Unix because the extension casing shouldn't matter there as well).